hassio/src/dashboard/hassio-update.ts

```diff
@@ -74,26 +64,26 @@ export class HassioUpdate extends LitElement {
         <div class="card-group">
           ${this._renderUpdateCard(
             "Home Assistant Core",
-            this.hassInfo!,
+            this.supervisor.core!,
```

Comment:
This could be undefined

Reply:
How?

Reply:
You are right, so we can drop the `!` :-)